src/components/forms/utils.js

```diff
@@ -0,0 +1,7 @@
+export const valueHandler = callback => (typeof callback === 'function' ? value => callback(value) : null);
+
+export const eventValueHandler = (callback, defaultValue = '') =>
+  typeof callback === 'function' ? event => callback(event.target.value || defaultValue) : null;
+
+export const checkboxHandler = (callback, defaultValue = false) =>
```

Comment:
does it makes sense to document these utils?

Reply:
I don't think that is necessary